code/src/components/Album.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Artist } from "./Artist";
+
+export const Album = (props) => {
+  // console.log(artistList);
```

Comment:
You can take these out completely once you're finished to make your code cleaner